src/app/services/forms/user-registration-form-service/user-registration-form.service.ts

```diff
@@ -8,7 +8,9 @@ import {I18nPagesValidationsEnum} from '@common/enums/i18n';
 import {FormsService} from '@services/general';
 import {IFormValidationError} from '@common/interfaces';
 
-@Injectable()
+@Injectable({
```

Comment:
Esto me va a romper logica,

Lo tenia como


```suggestion
@Injectable()
```

porque aproposito necesito que el ciclo de vida de este servicio este asociado al ciclo de vida del componente. No que este global.

Pruebe ejecutar el flujo de registro de usuario normal por el de google y se dara cuenta que van a existir comportamientos incorrectos.

Reply:
Probado y funcionando, no rompe la logica como lo hablamos